.travis.yml

```diff
@@ -2,11 +2,11 @@ language: python
 python:
   - 2.7
   - 3.6
-dist: trusty
+dist: focal
 matrix:
   include:
     - python: 3.7
-      dist: xenial
+      dist: focal
```

Comment:
This line is entirely unneeded. (It probably dates from when 3.7 wouldn't install on the old version used before Trusty…)

Reply:
Unnecessary lines removed, I think... ?